src/main/java/com/ksm/domino/cli/command/AbstractDominoCommand.java

```diff
@@ -125,8 +118,12 @@ public void output(Object o, Domino domino) throws JsonProcessingException {
 
     public String getRequiredParam(Map<String, String> parameters, String parameterName, String command) {
         String param = parameters.get(parameterName);
-        Validate.notBlank(param,
+        try {
+            Validate.notBlank(param,
                     String.format("Missing the required parameter '%s' when calling '%s'.", parameterName, command));
+        } catch (Exception e) {
+            throw new IllegalArgumentException(e.getMessage());
+        }
```

Comment:
I think instead of Validate.notBlank we can do...

```java
if (StringUtils.isBlank(param)) {
   throw new IllegalArgumentException(String.format("Missing the required parameter '%s' when calling '%s'.", parameterName, command));
}